.github/workflows/ci.yml

```diff
@@ -2,10 +2,10 @@ name: Continuous Integration testing
 on:
   push:
     branches:
-      - master
+      - feature-fuk95
```

Comment:
These should not be changed here (and are also why the checks have not run for this PR, I think)

Reply:
Thanks for pointing this out, @nordmoen! I keep changing the branch for CI testing while working on it and forget to change back. The CI test on ubuntu fails because fetching http://azure.archive.ubuntu.com/ubuntu/pool/main/n/nvidia-settings/libxnvctrl0_440.82-0ubuntu0.20.04.1_amd64.deb fails. I am not sure why the tests on macOS keep getting cancelled. There might still be a meson 0.57.1 issue on macOS, but the test seems to get cancelled before getting to this. Tests with Intel compilers have persistently passed.

Reply:
They are simply canceled because they are in the same `matrix` in Github action parlance. We can change the [`fail-fast` behavior](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast) of this setup so that this is not the case.